### DISCUSSION TITLE
Data formats: Displayed values change on client only or also on server?

### DISCUSSION THREAD

**Discussion by TheMechanist902** [dbeaver/dbeaver#15652] [Q&A]
Hi,
I'm new to dBeaver and DBMS in general. But since I must use dBeaver on a production database at work I would rather ask this before trying. Do the locale settings [here](https://dbeaver.com/docs/wiki/Managing-Data-Formats) just affecting the display of the values on the dBeaver client side? My guess is, that it works comparatively to Excel number formats: The displayed text changes, but the value behind it remains the same. Or are the setting actually changing something on the SQL Server? I would rather not not knowingly change something there. 
I would be glad if you could give me some clarity here. I guess for more experienced DBAs this is a rather trivial question, but any help would be greatly appreciated.
Thank you.

**Comment by LonwoLonwo** [comment]
Hello @TheMechanist902 

You can be calm. Changing the data display settings does not affect the data itself. To change the data from the DBeaver - we take these data obtained, switch them on our side, and after it - show you. DBeaver will not send any changes to your database if you do not ask for this specifically. 
You can check it with our tool - [Query Manager.](https://github.com/dbeaver/dbeaver/wiki/Query-Manager) 
Query Manager records the requests that DBeaver creates.

**Reply by TheMechanist902** [reply]
Hi @LonwoLonwo,
thank you for the quick and extensive reply. I never thought of the query manager as a helper in understanding these aspects. I will have to look into it more thoroughly in that regard. I guess I can learn quite a lot from it.
